### DISCUSSION TITLE
塑膠變丙烷的新技術！🛍️

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#782] [General]
新聞來源:
https://technews.tw/2022/10/11/plastics-recycling-propane/

內容大綱:
麻省理工團隊提出的新技術可以處理混合在一起的多種塑膠，將它們轉化為單一的產品──丙烷。
新型催化劑也適用多種類型的塑膠，包括最常用的聚乙烯（PET）和聚丙烯（PP）。實際測試混合塑膠實驗後，團隊也發現催化劑可以將大約 80% 的塑膠轉化為丙烷，也不會產生甲烷等副產品。生成的丙烷可直接當成低環境影響的燃料，或用在封閉循環系統中製造新塑膠原料。

心得感想:
文章中有提到這個技術的催化劑取得成本較低，不僅能讓塑膠的回收過程加速，也能讓製造的丙烷再次被利用，如果在未來能將技術普及，對環境的傷害可以大大的降低。希望這個技術在未來能讓更多國家跟政府參考引用，改善地球的現況！

**Comment by moonnight3503** [comment]
化學真是太神奇了，大幅減少對環境的傷害

**Reply by bonny123865** [reply]
化學神奇又神秘，我只需要知道結果就好
感覺要深入了過程，又是另外一回事了!!

**Comment by a45320614** [comment]
這個新技術如果真的能實際應用，對環保產業應該也會產生不小的衝擊，希望能改善目前台灣回收業的惡性循環，達成真正的回收再利用

**Reply by bonny123865** [reply]
我也覺得這項技術說不定能讓環保產業更精進現有的設備，然後達到這種效果!

**Comment by Pietra1226** [comment]
這樣對地球環境永續發展又更進一步了

**Comment by manipulateed** [comment]
材料科學確實也是一門為了地球發展的科系，相比將能帶給人類更大的福祉。

**Comment by Tuoresa** [comment]
人造材料完全的回收再利用真的是地球永續非常重要的一環，尤其塑膠這樣可能造成長時間危害的合成物，能在封閉循環中製造能一次解決民生必須用品及環保問題，著實是環保產業全新的一頁！

**Comment by delia07152** [comment]
雖然各個國家都在推出減塑的政策，但塑膠的使用率仍然很高，如果人們可以透過這項技術，讓塑膠轉化為丙烷，不但能解決塑膠的處理問題，還能解決能源問題！

**Comment by Kotori0501** [comment]
如果能運用此技術，就不用太擔心漂流海洋中塑膠被魚類吃掉了
